tests/test_selenium.py

```diff
@@ -42,9 +46,20 @@ def test_login():
 
     options = Options()
     options.add_argument("--headless")
-    with webdriver.Firefox(options=options) as driver:
+
+    profile = webdriver.FirefoxProfile()
+
+    # If testing against proxy'ed microk8s instance, uncomment these lines
```

Comment:
Is there no way of doing this by passing through some kind of argument or option, so this is not a manual step of uncommenting some lines? Or is it not worth the hassle?

Reply:
Yeah, you can do it with a `conftest.py` file.  Example: [adding options](https://github.com/juju-solutions/ops-reactive-interface/blob/main/tests/integration/conftest.py#L16-L21), and [using an option](https://github.com/juju-solutions/ops-reactive-interface/blob/main/tests/integration/conftest.py#L198).

Reply:
Upstream docs / example (probably doesn't even need a separate `conftest.py` file): https://docs.pytest.org/en/stable/example/simple.html#pass-different-values-to-a-test-function-depending-on-command-line-options